hack/build.sh

```diff
@@ -92,12 +92,13 @@ for dir in ${dirs}; do
   fi
 
   if [[ -v TEST_MODE ]]; then
-    VERSION=$dir IMAGE_NAME=${IMAGE_NAME} test/run
-
+    TEST_DIR=../hack
+    pushd $TEST_DIR && make $TEST_CASE VERSION=$dir IMAGE_NAME=${IMAGE_NAME}
```

Comment:
>The test/run script should be run from common.mk somehow (if someone wanted to make that pretty).

I agree this would be the best way to do it. Although I'm not sure how to do that without doing a bigger overhaul of `build.sh` and `common.mk`. Probably best to leave it for now and rewrite it as a part of a separate issue.